packages/stg-devtools-evm-hardhat-v2/src/blacklistable/mixin.ts

```diff
@@ -22,6 +22,11 @@ export const BlacklistableMixin: IBlacklistable = {
         return {
             ...this.createTransaction(data),
             description: `Setting blacklister to address ${blacklister}`,
+            metadata: {
+                contractName: this.contractName,
```

Comment:
Not recognized here, but recognized in sdk... looking into this

Reply:
maybe can pass as param

Reply:
or change parameter `this` to be `Blacklistable` instead of `OmniSDK`, but that may break the call from usdc to blacklistable [here](https://github.com/stargate-protocol/stargate-v2/blob/bc089aca096c8ca0cb167fce0b3f1225b431f68f/packages/stg-devtools-evm-hardhat-v2/src/usdc/sdk.ts#L52)